### PR TITLE
chore: move angular updates from Dependabot to Renovate

### PR DESCRIPTION
**Pull request by xiduzo**
Removed the frontend from the Dependabot github-action, added it to Renovate

Solves [PZ-4773](https://dimpact.atlassian.net/browse/PZ-4773)

### CODE REVIEW

**Review by infonl-marcel — APPROVED** [review]

File: renovate.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   "enabledManagers": [
```
Comment (niorg) [inline renovate.json:10]: No need to be explicit here, we can fully remove the `enabledManagers` and let renovate autodetect

File: renovate.json [hunk 2]
```diff
@@ -88,6 +89,72 @@
       ],
       "enabled": false,
       "description": "The version of the ZAC Docker image in our Docker Compose file is set to 'latest' and is only used for local development"
+    },
+    {
```
Comment (niorg) [inline renovate.json:93]: Renovate has quite a few grouping rules pre-specified. I'd suggest to see how useful these are in our setup and figure out how to tune it from there. 

Some other settings seem incorrect here, I guess you can just remove all these added PackageRules for now.

**Review by niorg — CHANGES_REQUESTED** [review]

File: renovate.json (hunk 2 above)
Comment (bas-info-nl) [reply renovate.json:87]: What about the rules defined for dependabot? Will we have to re-create those for renovate? Or do that after we get a request to update those dependencies (ie. angular, node, bootstrap) ?

**Review by bas-info-nl — APPROVED** [review]

File: renovate.json (hunk 2 above)
Comment (niorg) [reply renovate.json:87]: Better to do that afterwards. Let's first see what grouping it comes up with automatically and we can easily create new packagerules based on that

Comment (edgarvonk) [reply renovate.json:87]: Agreed. One the reasons we want to migrate to Renovate in the first place is that it is much more clever and has a lot of pre-defined rules. In my fork where I tested out Renovate some time ago I did not have to create any rules I think.

**Review by edgarvonk — COMMENTED** [review]
I though we had a story on the backlog to move to Renovate or not?

Also, we have our WebDAV project which is ideal to try out Renovate. There I would just remove Dependabot completely and use a default blank Renovate config to see how that goes.

**Comment by niorg** [comment]
> I though we had a story on the backlog to move to Renovate or not?
> 
> Also, we have our WebDAV project which is ideal to try out Renovate. There I would just remove Dependabot completely and use a default blank Renovate config to see how that goes.

Renovate was already enabled there, but I tuned the config a bit:
https://github.com/infonl/webdav-servlet/pull/113

Might be a good template for this renovate config too

**Pull request closed without merging**